tools/inventory.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+use Glpi\Inventory\Conf;
+use Glpi\Inventory\Request;
+
+$conf = new Conf();
+if ($conf->enabled_inventory != 1) {
+    die("Inventory is disabled");
+}
+
+if (!isCommandLine()) {
+    die('This script is only available from the command line');
+}
+
+$inventory_request = new Request();
+if ($inventory_request->inError() && $inventory_request->getHttpResponseCode() == 415) {
+    http_response_code($inventory_request->getHttpResponseCode());
+    die("Unsupported compression");
+}
+$inventory_request->handleHeaders();
```

Comment:
This one should have no effect in the CLI context.

Reply:
Indeed, I removed it